src/profile-logic/tracks.js

```diff
@@ -943,6 +1062,9 @@ export function computeDefaultVisibleThreads(
     return { threadIndex, score };
   });
 
+  // Next, filter the tracks by the tab selector threads.
+  scores = scores.filter(({ threadIndex }) => allTrackThreads.has(threadIndex));
```

Comment:
Instead of filtering as an afterthought, would it be possible to use the global and local tracks information directly and generate the scores only for those? Unless we'd like to cache the scores for a faster switch (if the scoring computation is slow, I don't know).

Reply:
I'm not super convinced but OK.